src/main/java/duke/DeadlineCommand.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+/**
+ *
+ * Command to add a Deadline Task to the existing list of tasks.
+ */
```

Comment:
There is no need for empty line before and after javadoc. 
```suggestion
/**
 * Command to add a Deadline Task to the existing list of tasks.
 */
```